bigbluebutton-html5/imports/ui/components/poll/component.jsx

```diff
@@ -393,6 +404,7 @@ class Poll extends Component {
             isMeteorConnected,
             stopPoll,
             currentPoll,
+            secretPoll: this.state.secretPoll,
```

Comment:
You need to pass the value from `currentPoll.secretPoll` here instead of state, otherwise when the presenter just closes and reopens the poll pannel, or refresh the page he can see the answers (it resets to false).

Reply:
Done in https://github.com/bigbluebutton/bigbluebutton/pull/12430/commits/19ca64859fe51fa8ffb8a45f0e37942a410ab88b